.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Added this file so that git does not track any changes in the package-lock.json file. This was because the .gitignore was not able to take care of this particular file.